linters/terrascan/plugin.yaml

```diff
@@ -16,15 +16,25 @@ lint:
     - name: terrascan
       download: terrascan
       known_good_version: 1.18.1
-      files: [terraform]
       suggest_if: config_present
       commands:
         - name: lint
           output: sarif
           is_security: true
           run: terrascan scan --iac-file ${target} --output sarif
+          read_output_from: stdout
+          files: [terraform] # terrascan supports - arm, cft, docker, helm, k8s, kustomize, terraform. However, we only support terraform and docker as filetypes.
           success_codes: [0, 3, 4, 5]
+          parser:
+            runtime: python
+            run: ${plugin}/linters/terrascan/sarif_to_sarif.py
```

Comment:
nit: I remember we added this before, but might be nice to add a comment as to why we do this. IIRC, it was because the output was too verbose for a normal diagnostic

Reply:
done